site/templates/footer/app-badges.jet

```diff
@@ -0,0 +1,13 @@
+{{ block appBadges() }}
```

Comment:
We went through the team bloodletting for the holy whitespace war, I feel it would be remiss of me not to request striking down the unholy spaces of Satan.